packages/base-types/src/node/randomV2.ts

```diff
@@ -0,0 +1,23 @@
+import { NodeType } from './constants';
```

Comment:
I think you also need to import and export this on the index.ts file in the same folder